third-party/cmake/BuildLuajit.cmake

```diff
@@ -76,57 +76,6 @@ if(UNIX)
     CC=${DEPS_C_COMPILER} PREFIX=${DEPS_INSTALL_DIR}
     ${DEPLOYMENT_TARGET})
 
-elseif(MINGW AND CMAKE_CROSSCOMPILING)
```

Comment:
This we can definitely get rid of. Not sure if we need crosscompiling stuff at all.